ja/development/testing.rst

```diff
@@ -1,6 +1,8 @@
 テスト
 #######
 
+.. todo:: (Japanese) このファイルは https://github.com/cakephp/docs/pull/7813 においてCIのエラーを回避するために修正されましたが、きちんと翻訳が更新された訳ではありません。翻訳を頑張りましょう。
+
```

Comment:
Thans for your comments 🤗 

in my opinion,

- todo comments are intended to manage tasks of authors and translators, because they are not visible on production site https://book.cakephp.org/ . (if it's wrong or seems to be wrong, please tell me)
- todo comments will help us to determine which file is the next translation target. Unfortunately, translators have no task management tools at this point. So, we will check `git commit` timestamp, todo comments and sometimes user's requests. In this pull request, `testing.rst` and `plugins.rst` are updated without complete translations. That's why I thought todo comments are effective.

Let me leave these todo comments as they are, if possible.